src/Compilers/CSharp/Portable/Symbols/NullableAnnotationExtensions.cs

```diff
@@ -52,5 +53,28 @@ internal static CodeAnalysis.NullableAnnotation ToPublicAnnotation(this Nullable
             Debug.Assert((CodeAnalysis.NullableAnnotation)(NullableAnnotation.Oblivious + 1) == CodeAnalysis.NullableAnnotation.Disabled);
             return (CodeAnalysis.NullableAnnotation)annotation + 1;
```

Comment:
>(CodeAnalysis.NullableAnnotation)annotation + 1 [](start = 19, length = 47)

Consider using a `switch` instead.

Reply:
I've been resisting doing this, but you and @jcouv have protested enough that I'll cave.

---
In reply to: [269209292](https://github.com/dotnet/roslyn/pull/34429#discussion_r269209292) [](ancestors = 269209292)